src/Core/src/Handlers/RefreshView/RefreshViewHandler.Android.cs

```diff
@@ -18,7 +18,7 @@ protected override void ConnectHandler(MauiSwipeRefreshLayout platformView)
 
 		void OnSwipeRefresh(object? sender, System.EventArgs e)
 		{
-			VirtualView.IsRefreshing = true;
+			((IElementController)VirtualView).SetValueFromRenderer(RefreshView.IsRefreshingProperty, true);
```

Comment:
Can you change this to work via interface on the `RefreshView`? 

Here's an example on `InputView`

Reply:
Yes I see.  Looks like this is already there as per #17062